db-scheduler/src/test/java/com/github/kagkarlsson/scheduler/task/DailyTest.java

```diff
@@ -5,47 +5,138 @@
+        // Europe/Rome is UTC+2 during daylight time
+        LocalDate currentDate = LocalDate.of(2020, 7, 12);
+        LocalDate nextDay = currentDate.plusDays(1);
+
+        assertThat(new Daily(ROME, HOUR_8).getNextExecutionTime(complete(instant(currentDate, HOUR_0, UTC))), is(instant(currentDate, HOUR_6, UTC)));
+        assertThat(new Daily(ROME, HOUR_8).getNextExecutionTime(complete(instant(currentDate, HOUR_6, UTC))), is(instant(nextDay, HOUR_6, UTC)));
+    }
+
+    @Test
+    public void should_generate_next_date_correctly_on_the_day_before_time_changes_to_daylight_saving() {
+        // Europe/Rome is UTC+1 during standard time
+        // Europe/Rome is UTC+2 during daylight saving time
+        // Daylight saving time begins in Rome on 29 March 2020
+        LocalDate currentDate = LocalDate.of(2020, 3, 28);
+        LocalDate nextDay = currentDate.plusDays(1);
+
+        assertThat(new Daily(ROME, HOUR_8).getNextExecutionTime(complete(instant(currentDate, HOUR_0, UTC))), is(instant(currentDate, HOUR_7, UTC)));
+        assertThat(new Daily(ROME, HOUR_8).getNextExecutionTime(complete(instant(currentDate, HOUR_7, UTC))), is(instant(nextDay, HOUR_6, UTC)));
+    }
+
+    @Test
+    public void should_generate_next_date_correctly_on_the_day_before_time_changes_to_standard() {
+        // Europe/Rome is UTC+1 during standard time
+        // Europe/Rome is UTC+2 during daylight saving time
+        // Standard time begins in Rome on 25 October 2020
+        LocalDate currentDate = LocalDate.of(2020, 10, 24);
+        LocalDate nextDay = currentDate.plusDays(1);
+
+        assertThat(new Daily(ROME, HOUR_8).getNextExecutionTime(complete(instant(currentDate, HOUR_0, UTC))), is(instant(currentDate, HOUR_6, UTC)));
+        assertThat(new Daily(ROME, HOUR_8).getNextExecutionTime(complete(instant(currentDate, HOUR_6, UTC))), is(instant(nextDay, HOUR_7, UTC)));
+    }
+
+    @Test
+    public void should_generate_next_date_correctly_on_the_day_when_time_changes_to_daylight_saving() {
+        // Europe/Rome is UTC+1 during standard time
+        // Europe/Rome is UTC+2 during daylight saving time
+        // Daylight saving begins on 29 March 2020 at 2AM
+        LocalDate currentDate = LocalDate.of(2020, 3, 29);
+        LocalDate nextDay = currentDate.plusDays(1);
+
+        assertThat(new Daily(ROME, HOUR_2).getNextExecutionTime(complete(instant(currentDate, HOUR_0, UTC))), is(instant(currentDate, HOUR_1, UTC)));
+        assertThat(new Daily(ROME, HOUR_3).getNextExecutionTime(complete(instant(currentDate, HOUR_0, UTC))), is(instant(currentDate, HOUR_1, UTC)));
+    }
+
+    @Test
+    public void should_generate_next_date_correctly_on_the_day_when_time_changes_to_standard() {
+        // Europe/Rome is UTC+1 during standard time
+        // Europe/Rome is UTC+2 during daylight saving time
+        // Standard saving begins on 25 October 2020 at 3AM
+        LocalDate currentDate = LocalDate.of(2020, 3, 29);
```

Comment:
Did you mean to use october here?

Reply:
Yup you are absolutely right :relaxed:, fixed.